FINAL_PROJECT/`R/hw04.Rmd

```diff
@@ -33,49 +33,45 @@ adhd_data <- read_csv(here::here("FINAL_PROJECT","data","adhd.csv"),
 
 adhd_data %>%
   group_by(Ethnicity, Measure, Score) %>%
-  summarise(across(c(Value),
-            list(
-              mean = ~mean(.x, na.rm = TRUE),
-              sd = ~sd(.x, na.rm = TRUE), 
-              min = ~min(.x, na.rm = TRUE),
-              max = ~max(.x, na.rm = TRUE)
-              
-            )))
-
+  summarise(across(
+    c(Value),
+    list(
+      mean = ~ mean(.x, na.rm = TRUE),
+      sd = ~ sd(.x, na.rm = TRUE),
+      min = ~ min(.x, na.rm = TRUE),
+      max = ~ max(.x, na.rm = TRUE)
+    )
+  ))
 ```
 
-
 # Plots 
 
-
 ```{r, practicing plots}
 
 plot_adhd <- adhd_data %>%
-  ggplot()+
-  aes(x = Ethnicity,
-      y = Value) +
-  geom_point(fill = "purple" , 
-             color = "purple", 
-             alpha = .5,
-             size = 2,
-             shape = 20) +
+  ggplot() +
+  aes(
+    x = Ethnicity,
+    y = Value
+  ) +
+  geom_point(
+    fill = "purple",
+    color = "purple",
+    alpha = .5,
+    size = 2,
+    shape = 20
+  ) +
```

Comment:
Nothing wrong with how you style. This is the `tidyverse` style incase you were curious